site/en/tutorials/curl_quickstart.ipynb

```diff
@@ -0,0 +1,355 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !curl  -H 'Content-Type: application/json' \
</pre><p>You might consider giving the URL as the first arg here, like you do in the other examples -- just to lower cognitive load.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>